src/components/DateRangePicker.js

```diff
@@ -273,12 +285,22 @@ class DateRangePicker extends React.Component {
     const isLargeOrMediumWindowSize = this._isLargeOrMediumWindowSize(theme);
     const styles = this.styles(theme, isLargeOrMediumWindowSize);
     const shouldShowCalendarIcon = StyleUtils.getWindowSize(theme.BreakPoints) !== 'small';
+    const showCalendar = isLargeOrMediumWindowSize || this.state.showCalendar;
+    const { selectedEndDate, selectedStartDate } = this.state;
+
+    const mergedFocusTrapProps = {
+      focusTrapOptions: {
+        clickOutsideDeactivates: true
+      },
+      paused: false,
+      ...this.props.focusTrapProps
+    };
 
     return (
       <div style={styles.component}>
         <a
           onClick={this._toggleSelectionPane}
-          onKeyUp={(e) => keycode(e) === 'enter' && this._toggleSelectionPane()}
+          onKeyDown={(e) => keycode(e) === 'enter' && this._toggleSelectionPane()}
```

Comment:
Why the change from keyup to keydown?

Reply:
I'm not 100% sure why Luke did this but I would guess he was fixing an event listener over lap with the cancel/apply buttons.  They have key down listeners so I'm assuming the key up here was firing immediatley upon using one of those buttons. 

If you're opposed to this I could switch it back and then preventDefault/stop propagation on the buttons.

Reply:
It's fine how it is, I was just curious.